plugin/src/test/com/exadel/aem/toolkit/test/widget/AutocompleteWidget.java

```diff
@@ -0,0 +1,64 @@
+
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.test.widget;
+
+import com.exadel.aem.toolkit.api.annotations.main.AemComponent;
+import com.exadel.aem.toolkit.api.annotations.main.Dialog;
+import com.exadel.aem.toolkit.api.annotations.widgets.DialogField;
+import com.exadel.aem.toolkit.api.annotations.widgets.autocompletecoral3.Autocomplete;
+import com.exadel.aem.toolkit.api.annotations.widgets.autocompletecoral3.AutocompleteOption;
+import com.exadel.aem.toolkit.api.annotations.widgets.common.OptionProvider;
+import com.exadel.aem.toolkit.api.annotations.widgets.common.OptionSource;
+import com.exadel.aem.toolkit.api.annotations.widgets.select.Select;
+
+import static com.exadel.aem.toolkit.plugin.utils.TestConstants.DEFAULT_COMPONENT_NAME;
+
+@AemComponent(
+    path = DEFAULT_COMPONENT_NAME,
+    title = "Autocomplete Widget Dialog"
```

Comment:
... ~Dialog~. It's the title of a component

Reply:
done